### PR TITLE
Add chrome-go Dockerfile

### PR DESCRIPTION
**Pull request by p-patel**
Add Dockerfile for new chrome-go image which will be used to run UI components tests

### CODE REVIEW

File: chrome-go/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM golang:1.17.4
+
+RUN wget -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
```
Comment (dandwal) [inline chrome-go/Dockerfile:3]: We should[ try to limit the number of RUN commands](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers), each RUN command creates a new layer, the echo into the file probably doesn't warrant one maybe we merge them into one item?
```suggestion
RUN wget -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
  && echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list
```

Comment (p-patel) [reply chrome-go/Dockerfile]: Yeah good point, updated

**Review by dandwal — DISMISSED** [review]
LGTM

**Comment by p-patel** [comment]
Signed the commit

**Review by dandwal — APPROVED** [review]

**Pull request merged**